scripts/BUILD.bazel

```diff
@@ -118,6 +118,14 @@ kt_jvm_binary(
     runtime_deps = ["//scripts/src/java/org/oppia/android/scripts/maven:generate_maven_dependencies_list_lib"],
 )
 
+kt_jvm_binary(
+    name = "generate_license_texts",
```

Comment:
Maybe instead: ``retrieve_license_texts`` or similar? I think my earlier comment about 'generation' applies here, too. Ditto for all the library/class names elsewhere.

Reply:
Done.